NitroxPatcher/Patches/Dynamic/FootstepSounds_OnStep_Patch.cs

```diff
@@ -61,4 +74,27 @@ private static float CalculateVolume(float originalVolume, FootstepSounds instan
         Resolve<FMODWhitelist>().TryGetSoundData(EXO_STEP_SOUND_PATH, out SoundData soundData);
         return soundData.Radius;
     });
+    public static void SendFootstepPacket(FMODAsset asset)
+    {
+        // Send footstep packet with the asset
+        var localPlayer = Resolve<LocalPlayer>();
```

Comment:
You don't need to cache `Resolve<T>()`. It's already cached under the hood and can be used/called multiple times as it is without performance decreases.

Reply:
Done